app/Main.hs

```diff
@@ -234,6 +245,7 @@ main = do
     execute CommandOptions {dockerfiles = []} =
       putStrLn "Please provide a Dockerfile" >> exitFailure
     execute cmd = do
+      when (showConfigFile cmd) (putStrLn $ getFilePathDescription (configFile cmd))
```

Comment:
I think this should got to stderr, instead of stdout, so that the parsing of error messages is not disrupted

Reply:
sure, this sounds good! could i just get extra clarification on the parsing of error messages and how it's being done right now?

Reply:
Oh, that is being done by third parties, like editors and CI systems

Reply:
Ah I see. Thank you